packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -319,6 +350,39 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     sessionStorage?.setItem(GENERATED_ANSWER_DATA_KEY, JSON.stringify(data));
   }
 
+  /**
+   * Returns the generated answer element.
+   * @returns {HTMLElement}
+   */
+  get generatedAnswerElement() {
+    return this.template.querySelector('.generated-answer__answer');
+  }
+
+  /**
+   * Returns the generated answer height.
+   * @returns {number}
+   */
+  get generatedAnswerElementHeight() {
+    // @ts-ignore
+    return getAbsoluteHeight(this.generatedAnswerElement?.firstChild);
```

Comment:
we need this ts-ignore?

Reply:
There's a bunch of conflicting types here, `firstChile` returns a `ChildNode` instead of an HTMLElement but it doesn't matter etc.